src/main/webapp/i18n/en/singleRecordView.json

```diff
@@ -5,7 +5,14 @@
       "otherTabs": "Other tabs..."
     },
     "details": {
-      "title": "Organization details"
+      "title": "Organization details",
+      "compare": {
+        "title": "Compare",
+        "header": " partners have similar records",
+        "content1": "Reviewing these records will help us give you better recommendation in the future.",
+        "content2": "A manual review is the easiest way to confirm that these records match",
```

Comment:
I think there should be a period at the end of a sentence.

Reply:
true